cmd/merge/main.go

```diff
@@ -45,8 +45,9 @@ func (fm flagMap) String() string {
 	for key, value := range fm {
 		if value == "" {
 			parts = append(parts, key)
+		} else {
```

Comment:
nit: I'd probably write this as

```golang
if value == "" {
  parts = foo()
  continue
}
parts = foo()
```

(to always try and keep indentation minimized)